lib/ui/views/about/about_tos_view.dart

```diff
@@ -0,0 +1,235 @@
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+class AboutTosView extends StatelessWidget {
+  static const String id = 'about_tos_view';
+
+  final bool showAppBar;
+
+  const AboutTosView({Key key, this.showAppBar = true}) : super(key: key);
+
+  TextSpan _buildText(BuildContext context, String text, {bool bold = false}) {
+    return TextSpan(
+        style: Theme.of(context)
+            .textTheme
+            .bodyText1
+            .copyWith(fontWeight: bold ? FontWeight.bold : FontWeight.normal),
+        text: text);
+  }
+
+  TextSpan _buildLink(String text, String link) {
+    return TextSpan(
+      style: TextStyle(color: Colors.blue),
+      text: text,
+      recognizer: TapGestureRecognizer()
+        ..onTap = () async {
+          final url = link;
+          if (await canLaunch(url)) {
+            await launch(
+              url,
+              forceSafariVC: false,
+            );
+          }
+        },
+    );
+  }
+
+  Widget _buildSection(
+      {@required BuildContext context,
+      @required String title,
+      @required List<TextSpan> content}) {
+    return Container(
+      child: Column(children: [
+        Text(
+          title,
+          style: Theme.of(context)
+              .textTheme
+              .headline5
+              .copyWith(fontWeight: FontWeight.w400, color: Colors.black),
+          textAlign: TextAlign.left,
+        ),
+        RichText(
+          textAlign: TextAlign.justify,
+          text: TextSpan(
+            children: content,
+          ),
+        )
+      ]),
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: showAppBar ? AppBar(title: Text('Terms of Service')) : null,
+      body: SingleChildScrollView(
+        padding: const EdgeInsets.all(16),
+        child: Column(
+          children: <Widget>[
+            _buildSection(

```

Comment:
Put the subheadings like 1.1 **aligned to the left** and make them consistent with others.

Reply:
![image](https://user-images.githubusercontent.com/22657113/99490479-f147aa80-298f-11eb-8039-2f7a3fb1596a.png)
Done!